docs/products/postgresql/howto/use-pg-repack-extension.rst

```diff
@@ -0,0 +1,20 @@
+Use the PostgreSQL® ``pg_repack`` extension
+===========================================
+
+``pg_repack`` is a PostgreSQL extension that allows you to efficiently reorganize tables to remove any excess bloat the tables have accumulated.  
+
+.. note:: 
+  Reorganizing a table may take some time, but ``pg_repack`` tries to minimize the locks required to continue online operations.
+
+To use the ``pg_repack`` extension: 
+
+1. Run the following command as ``avnadmin`` user: 
```

Comment:
You need to connect to the database first, we can either say "While connected to the database as ``avnadmin`` user, run the following command:"